packages/maker/msix/src/MakerMSIX.ts

```diff
@@ -0,0 +1,39 @@
+import { MakerBase, MakerOptions } from '@electron-forge/maker-base';
+import { ForgePlatform } from '@electron-forge/shared-types';
+import { packageMSIX } from 'electron-windows-msix';
```

Comment:
If this is an optional dependency should this be lazily imported? Or should it be non-optional?